tsconfig.json

```diff
@@ -1,5 +1,47 @@
 {
   "compilerOptions": {
+    /* Visit https://aka.ms/tsconfig to read more about this file */
```

Comment:
suggestion: this link redirects to `https://www.typescriptlang.org/tsconfig`, can we use this link directly?